graphite_api/render/glyph.py

```diff
@@ -12,7 +12,12 @@
 See the License for the specific language governing permissions and
 limitations under the License."""
 
-import cairocffi as cairo
+try:
+    import cairocffi as cairo
+except (NameError, ImportError, AttributeError):
```

Comment:
`except ImportError:` should be enough. What triggered NameError / AttributeError?